### DISCUSSION TITLE
hull volume hard to read

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#10895] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

hull volume is a bit hard to read due to no separators

![image](https://user-images.githubusercontent.com/5103517/211189164-2dad1d00-58be-4446-b732-cbdd21efa300.png)

### Reproduction steps

select 1 million+ volume of hulls

### Bug prevalence

Happens regularly

### Version

0.20.16.1

### Which operating system did you encounter this bug on?

Windows

**Comment by Prolbo** [comment]
Yes, having separators, simple spaces or commas, would be nice.
